components/formats-gpl/pom.xml

```diff
@@ -65,8 +65,13 @@
     </dependency>
     <dependency>
       <groupId>edu.ucar</groupId>
-      <artifactId>netcdf</artifactId>
-      <version>4.3.19</version>
+      <artifactId>netcdf4</artifactId>
+      <version>4.6.5</version>
+    </dependency>
+    <dependency>
+      <groupId>edu.ucar</groupId>
+      <artifactId>cdm</artifactId>
```

Comment:
I would remove this dependency declaration as it should be resolved as part of the `netcdf4` 4.6.5 dependencies and have the exclusions associated to the `netcdf4` artifact.